module-2/backend/week_two.md

```diff
@@ -5,28 +5,91 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+<br>
+ActiveRecord seperates the user from sql allowing them to work with sql databases in a more user friendly and readable manner.
+<br>
 2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+<br>
```

Comment:
This is a stale version of the questions. Be sure to set the Turing repo up as an [upstream repository](https://help.github.com/articles/configuring-a-remote-for-a-fork/), and then [fetch the latest version](https://help.github.com/articles/syncing-a-fork/) before you start working on the questions for the week.